scipy/signal/tests/test_filter_design.py

```diff
@@ -1772,6 +1790,15 @@ def test_invalid_input(self):
             cheb2ord([0.1, 0.6], [0.2, 0.5], 1, -2)
         assert "gstop should be larger than 0.0" in str(exc_info.value)
 
+    def test_ellip_butter(self):
+        # The purpose of the test is to make sure the result of `cheb2ord`
+        # differs from that of `buttord`. The values to compare to are
+        # generated with scipy 1.9.1
```

Comment:
minor: should all these added tests have the same name? `cheb2ord` is being compared to `buttord` but `ellip` in still in the test name? Not sure if intentional or copy/paste thing?

Also, was `1.9.1` chosen as a reference just because you happen to have it installed or?

Reply:
Copy-paste error, yes. Fixed, thanks!

Re scipy version: absolutely, 1.9.1 is what I had installed at the time, and these things have not been touched for quite a while, so a specific version does not matter all that much I guess.
I've tweaked the comment, too.